src/Advisor/Advisor/ChangeLog.md

```diff
@@ -18,6 +18,7 @@
         - Additional information about change #1
 -->
 ## Upcoming Release
+* Implemented code refactoring, no behavior changes expected.
```

Comment:
Suggest to add the changelog as below:
`
Regenerate the module using the latest v4 generator to incorporate the new features added in the v4 generator.
`